examples/point-cloud-laz/worker/laslaz.js

```diff
@@ -0,0 +1,368 @@
+/* eslint no-use-before-define: 0 */
```

Comment:
plas.io, where the decoder originally comes from, did not publish it as a module.
we can polish the code, address the source and put it on npm for sure.
but it is going to take awhile. shall we hold this PR for that?